src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/BooleanExpressionComplexityCheck.java

```diff
@@ -74,6 +74,22 @@
  * <pre>
  * &lt;module name="BooleanExpressionComplexity"/&gt;
  * </pre>
+ * <p>Code Example:</p>
+ * <div class="wrapper">
+ * <pre>
+ * public class Test
+ * {
+ *  public static void main(String args[])
```

Comment:
Please change `String args[]` -> `String... args` to make it less c-stylish.

Reply:
Done